src/app/components/SubPageCard/index.tsx

```diff
@@ -14,6 +14,10 @@ type StyledComponentProps = {
   title?: ReactNode
   subheader?: string
   action?: ReactNode
+  /**
+   * An optional second title which will be displayed under title / subheader / action
+   */
+  title2?: ReactNode
```

Comment:
`subtitle` maybe?

Reply:
The problem is that currently we have `title`, `subheader` (and `action`), all displayed in the same row.
If we introduced `subtitle`, it would be very easy to accidentally confuse it with `subheader`.

That was my thinking, but I don't have a strong opinion. If you think subtitle would be better, I am open to rename it. What do you think?